src/vue-pdf-embed.vue

```diff
@@ -4,12 +4,17 @@
       v-for="pageNum in pageNums"
       :key="pageNum"
       :id="id && `${id}-${pageNum}`"
+      class="vue-pdf-embed__pages"
     >
       <canvas />
 
       <div v-if="!disableTextLayer" class="textLayer" />
 
       <div v-if="!disableAnnotationLayer" class="annotationLayer" />
+
+      <div>
+        <slot name="page" :page="pageNum"></slot>
```

Comment:
I think it would be more complete to have 2 slots (e.g. `prepage` and `postpage`, maybe you could come with some better names). What do you think?

Reply:
I agree. I went ahead and made an update. I used `before-page` and `after-page` since it seemd more descriptive, albeit more verbose. But i think thats okay.